enterprise-ws-spec/src/main/asciidoc/Handlers.adoc

```diff
@@ -125,146 +110,95 @@ Common scenarios are logging, metering, and accounting.
-must be maintained
+jakarta.jws.HandlerChain annotation. The ordering of the Handlers in the
+HandlerChain must follow the requirements specified in Jakarta XML Web Services
+specification. In addition to this, the ordering of any given type of Handler 
+(logical or protocol) in the deployment descriptor must be maintained
 
-A Web Services for Java EE provider is required to provide an
+A Web Services for Jakarta EE provider is required to provide an
 implementation of MessageContext.
 
-A Web Services for Java EE provider is required to provide all the
-interfaces of the javax.xml.rpc.handler.soap and
-javax.xml.ws.handler.soap package. The provider must also provide an
-implementation of the SOAPMessageContext interface.
+A Web Services for Jakarta EE provider is required to provide all the
+interfaces of the jakarta.xml.ws.handler.soap package. The provider must 
+also provide an implementation of the SOAPMessageContext interface.
 
 The programming model of a Port component can be single-threaded or
 multi-threaded as defined in sections link:#anchor-45[5.3.2.3] and
-link:#anchor-44[5.3.2.4]. The concurrency of a JAX-RPC or JAX-WS Handler
-must match the concurrency of the business logic it is associated with.
-Client handlers may need to support multi-threaded execution depending
-on the business logic which is accessing the Port.
+link:#anchor-44[5.3.2.4]. The concurrency of a Jakarta XML Web Services 
+Handler must match the concurrency of the business logic it is associated 
+with.  Client handlers may need to support multi-threaded execution 
+depending on the business logic which is accessing the Port.
 
 Handlers must be loaded using the same class loader the application code
 was loaded with. The class loading rules follow the rules defined for
 the container the Handler is running in.
 
-In a product that also supports JSR-299, an implementation must support
-use of 299-style managed beans as JAX-WS handler classes in an
-application. JSR-299 specifies the requirements for these
-container-managed bean instances w.r.t instantiation, injection and
-other services. JSR-299 defines @Dependent pseudo-scope, handler classes
-must be specified in that scope. It is an error if the handler class has
-a scope other than the required one.
+In a product that also supports Jakarta Contexts and Dependency Injection, 
+an implementation must support use of 299-style managed beans as Jakarta 
+XML Web Services handler classes in an application. Jakarta Contexts and 
+Dependency Injection specifies the requirements for these container-managed 
+bean instances w.r.t instantiation, injection and other services. 
+Jakarta Contexts and Dependency Injection specification defines @Dependent pseudo-scope, 
+handler classes must be specified in that scope. It is an error if the 
+handler class has a scope other than the required one.
 
 In a product that also supports Managed Beans, an implementation must
```

Comment:
Managed beans -> Jakarta Managed Beans

Reply:
Will update Managed Beans to Jakarta Managed Beans